DotNetRu.Commune.WasmClient/Program.cs

```diff
@@ -10,9 +15,30 @@ public static async Task Main(string[] args)
             var builder = WebAssemblyHostBuilder.CreateDefault(args);
             builder.RootComponents.Add<App>("#app");
 
-            builder.Services.AddBizLogic();
+            SetupLogging(builder.Logging, builder.Configuration);
```

Comment:
Мне не очень нравится использовать `Setup` когда стандартный FW использует `Configure`. Давай переименуем в `ConfigureLogging`
```suggestion
            ConfigureLogging(builder.Logging, builder.Configuration);
```

Reply:
Ок! Принято